Form/Extension/DateTypeExtension.php

```diff
@@ -59,7 +59,7 @@ public function finishView(FormView $view, FormInterface $form, array $options)
     /**
      * {@inheritdoc}
      *
-     * @todo Remove it when bumping requirements to SF 2.7+
+     * @deprecated Remove it when bumping requirements to SF 2.7+
```

Comment:
Just for curiosity: Why `@deprecated` instead of `@todo`? This will not trigger error on project using Symfony 2.7+?